neuralop/training/__init__.py

```diff
@@ -1,4 +1,6 @@
 from .losses import LpLoss, H1Loss
 from .trainer import Trainer
 from .torch_setup import setup
-from .callbacks import Callback, MGPatchingCallback, OutputEncoderCallback, SimpleWandBLoggerCallback, ModelCheckpointCallback
\ No newline at end of file
+from .callbacks import Callback, MGPatchingCallback, \
```

Comment:
Just use parenthesis instead of `\` :
```python
from .callbacks import (Callback, MGPatchingCallback, 
                                        ....)
````